MyXamarinApp.API/Repository/EmployeeRepository.cs

```diff
@@ -39,17 +74,31 @@ public async Task<EmployeeModel> AddEmployee(EmployeeModel employeeModel)
             var employee = _mapper.Map<Employee>(employeeModel);
             _context.Employees.Add(employee);
             await _context.SaveChangesAsync();
-            return employeeModel;
+            return await GetEmployeeById(employee.EmployeeGuid);
```

Comment:
Okay this is completely incorrect. I can pass any guid (existing or invalid guid) and the Add will create an employee with a different guid, because add discards the guid being sent and adds it on the db side ?
Make sure to connect with me on this